docs/preview/features/powershell/azure-logic-apps.md

```diff
@@ -93,29 +94,155 @@ PS> Disable-AzLogicAppsFromConfig -DeployFilename "./deploy-orderControl" -Resou
+# > Executing StopType 'Immediate' for Logic App 'rcv-shopping-order-file' in resource group 'my-resource-group'
 # Attempting to disable rcv-shopping-order-file
 # Successfully disabled rcv-shopping-order-file
 
-# > Executing StopType 'None' for Logic App 'rcv-shopping-order' in resource group 'my-resource-group'
+# -> Batch: 'Protocol Receiver(s)' has been executed
+
+# Executing batch: 'Generic Receiver(s)'
+# ==========================
+# > Executing StopType 'Immediate' for Logic App 'rcv-shopping-order' in resource group 'my-resource-group'
 # Attempting to disable rcv-shopping-order
 # Successfully disabled rcv-shopping-order
 
-# > Executing StopType 'None' for Logic App 'orc-shopping-order-processing' in resource group 'my-resource-group'
+# -> Batch: 'Generic Receiver(s)' has been executed
+
+# Executing batch: 'Orchestrator(s)'
+# ==========================
+# > Executing StopType 'Immediate' for Logic App 'orc-shopping-order-processing' in resource group 'my-resource-group'
 # Attempting to disable orc-shopping-order-processing
 # Successfully disabled orc-shopping-order-processing
 
-# > Executing StopType 'None' for Logic App 'snd-shopping-order-confirmation-smtp' in resource group 'my-resource-group'
+# -> Batch: 'Orchestrator(s)' has been executed
+
+# Executing batch: 'Sender(s)'
+# ==========================
+# > Executing StopType 'Immediate' for Logic App 'snd-shopping-order-confirmation-smtp' in resource group 'my-resource-group'
 # Attempting to disable snd-shopping-order-confirmation-smtp
 # Successfully disabled snd-shopping-order-confirmation-smtp
 
+# -> Batch: 'Sender(s)' has been executed
+```
+
+
+## Enabling Azure Logic Apps from configuration file  
+
+Typically done as the last task of the release pipeline, right after the deployment of the Logic Apps, as this will enable all specified Logic Apps in a specific order. 
+The Azure Logic Apps to be enabled and the order in which this will be done, will be defined in the provided configuration file.
+The order of the Azure Logic Apps in the configuration file (top to bottom) defines the order in which they will be enabled by the script. The counterpart of this script used to disable the Azure Logic Apps, will take the reversed order as specified (bottom to top) in the file.
+
+| Parameter         | Mandatory | Description                                                                                                         |
+| ----------------- | --------- | ------------------------------------------------------------------------------------------------------------------- |
+| ResourceGroupName | yes       | The resource group containing the Azure Logic Apps.                                                                 |
+| DeployFileName    | no        | If your solution consists of multiple interfaces, you can specify the flow-specific name of the configuration file. |
```

Comment:
```suggestion
| Parameter         | Mandatory | Description                                                                                                           |
| ----------------- | --------- | --------------------------------------------------------------------------------------------------------------------- |
| `ResourceGroupName` | yes       | The resource group containing the Azure Logic Apps.                                                                   |
| `DeployFileName`    | no        | If your solution consists of multiple interfaces, you can specify the flow-specific name of the configuration file.   |
```

Reply:
Shame this suggestion isn't part of the merge; as it makes sure that we have consistent docs.